CMakeLists.txt

```diff
@@ -34,42 +28,20 @@ if(POLICY CMP0012)
   cmake_policy(SET CMP0012 NEW)
 endif()
 
-find_package(OpenMP)
+if(OPENMP)
+  find_package(OpenMP REQUIRED COMPONENTS Fortran)
+endif()
+
 find_package(LAPACK)
 find_package(Jasper REQUIRED)
 find_package(PNG REQUIRED)
 find_package(ZLIB REQUIRED)
 
-#Prefer config files, and if not available use the regular FindXXX file
-set(CMAKE_FIND_PACKAGE_PREFER_CONFIG true)
-if(NOT TARGET w3nco_4)
-  find_package(W3NCO REQUIRED)
-endif()
+find_package(w3nco REQUIRED)
```

Comment:
On my laptop I get this error:
```
CMake Error at /usr/share/cmake-3.16/Modules/FindPackageHandleStandardArgs.cmake:146 (message):
  Could NOT find w3nco (missing: w3nco_path_d)
Call Stack (most recent call first):
  /usr/share/cmake-3.16/Modules/FindPackageHandleStandardArgs.cmake:393 (_FPHSA_FAILURE_MESSAGE)          
  cmake/Modules/Findw3nco.cmake:30 (find_package_handle_standard_args)
  CMakeLists.txt:40 (find_package)
```
I'm using new version of w3nco which is using cmake package config. The ``cmake/Modules/Findw3nco.cmake`` expects 'old style' environment variables to be set.

Reply:
Ah, I didn't have the modules checked out. We can just get rid it, right? It doesn't need NetCDF and that's the only one we rely on.

Reply:
In fact, I think none of the Find*.cmake modules from CMakeModules is needed.

Reply:
Yes, that's what I meant. The only thing we still use from that repo is `FindNetCDF`, which this doesn't use.

Reply:
Done

Reply:
I think you can delete ``.gitmodules``

Reply:
Done